polyfills/document.querySelector/polyfill.js

```diff
@@ -4,7 +4,7 @@
 
 	function getElementsByQuery(node, selector, one) {
 		var
-		generator = document.createElement(),
+		generator = document.createElement('null'),
```

Comment:
Sure, just wondering if that might be more likely to run into unknown browser bugs.  Not a strong point.